transform/src/apis.eta

```diff
@@ -20,7 +27,16 @@ export const apiConfigurations: Array<APIConfiguration> = [
         displayName: '<%= api.app.name %>',
         description: '<%= api.app.description %>',
         icon: '<%~ E.includeFile('icon', {icon: api.app.icon}) %>',
-        getApi: () => import('./apis/<%=api.path.join('/')%>/openapi.json') as unknown as Promise<OpenAPIV3.Document>
+        getApi: () => import('./apis/<%=api.path.join('/')%>/openapi.json') as unknown as Promise<OpenAPIV3.Document>,
```

Comment:
I found this really cool. Wasn't sure exactly what was happening in the string which led me to learn about EJS. Nice!